packages/aws-cdk-lib/custom-resources/lib/aws-custom-resource/aws-custom-resource.ts

```diff
@@ -394,6 +394,16 @@ export interface AwsCustomResourceProps {
    * @default - the Vpc default strategy if not specified
    */
   readonly vpcSubnets?: ec2.SubnetSelection;
+
+  /**
+   * Whether or not to include API response data in the logs for the underlying lambda.
+   *
+   * If this value is false, the raw API response and the `Data` field in the response
```

Comment:
I think we should add an example of `Response Object` here so that we explicitly mention what we will still be logging with this. And also add this to our documentation

Reply:
Updated.